gnovm/pkg/gnolang/machine.go

```diff
@@ -41,9 +43,10 @@ type Machine struct {
 	ReadOnly   bool
 	MaxCycles  int64
 
-	Output  io.Writer
-	Store   Store
-	Context interface{}
+	Output     io.Writer
+	Store      Store
+	Context    interface{}
+	VMGasMeter store.GasMeter
```

Comment:
Nit: VM seems redundant (this is the `Machine` struct), can `GasMeter` work instead?